index.html

```diff
@@ -1,14 +1,48 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Hello Microverse</title>
+    <title>Temesgen Portfolio</title>
     <link rel="stylesheet" href="style.css">
 
 </head>
+
 <body>
-    <h1 class="main-header">Hello Microverse!</h1>
+    <header>
+        <nav>
+            <a href="">
+                <h4>My Logo</h4>
+            </a>
+            <img src="./img/menu.png" alt="web menu graphics">
+        </nav>
+    </header>
+
+    <main>
+        <section class="Profile">
+            <article>
+                <h1>I’m Temesgen <br> Glad to see you!</h1>
+                <p>
+                    I’m a software developer! I can help you
+                    build a product, feature or website. Look
+                    through some of my work and experirience!
+                    If you like what you see and have a project
+                    you need coded, don't hestitate to contact me.
+                </p>
+            </article>
+            <h4>Lets connect</h4>
+            <ul class="social-handle">
+                <li><a href="#"><img src="./img/twitter.png" alt="twitter icon"></a></li>
+                <li><a href="#"><img src="./img/Linkedin.png" alt="linkedin icon"></a></li>
+                <li><a href="#"><img src="./img/medium.png" alt="medium icon"></a></li>
+                <li><a href="#"><img src="./img/github.png" alt="github icon"></a></li>
+                <li><a href="#"><img src="./img/angellist.png" alt="angellist icon"></a></li>
+            </ul>
```

Comment:
- [ ] Please you shoud do your best to match the project requirements .so try to match the design in all cases like your social icons are not correctly positioned(spaced), fonts don't match with Figma design, margins, padding, etc. kindly stick to design as much as possible you can do so by visualizing figma properties of some components of the design like colors,margins ,etc.


figma design:


![image](https://user-images.githubusercontent.com/90524466/170819354-1c6f489c-2908-49dc-a489-424468fff176.png)


your design:

![image](https://user-images.githubusercontent.com/90524466/170819382-dbe6cf57-af15-4612-9da9-96043bef6937.png)